cookbooks/bcpc-hadoop/recipes/default.rb

```diff
@@ -31,6 +31,11 @@
     'docopy' => false
 }
 
+node.default['bcpc']['hadoop']['copylog']['cheflog'] = {
+    'logfile' => "/var/log/chef/client.log",
```

Comment:
This should be an attribute of the Chef-Client cookbook [file name](https://github.com/chef-cookbooks/chef-client/blob/2d09db1fd44f1be536034311dd853284133e1e75/attributes/default.rb#L38) and [file path](https://github.com/chef-cookbooks/chef-client/blob/2d09db1fd44f1be536034311dd853284133e1e75/attributes/default.rb#L46).